modules/wine-deps.yml

```diff
@@ -0,0 +1,136 @@
+name: wine-deps
+buildsystem: simple
+build-commands: []
+x-compat32-module: .compat32-module.yml
+modules:
+
+  - name: openldap
+    config-opts:
+      - --disable-static
+      - --disable-slapd
+      - --disable-slurpd
+      - --disable-bdb
+      - --disable-hdb
+    cleanup:
+      - /bin
+      
+    sources:
+      - type: archive
+        url: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.47.tgz"
+        sha256: f54c5877865233d9ada77c60c0f69b3e0bfd8b1b55889504c650047cc305520b
+
+  - name: OpenCL
+    buildsystem: simple
+    x-multilib: false
+    build-commands:
+      - cp -av CL $FLATPAK_DEST/include/
+    sources:
+      - type: git
+        url: "https://github.com/KhronosGroup/OpenCL-Headers.git"
+        commit: c5a4bbeabb10d8ed3d1c651b93aa31737bc473dd
+
+  - name: libpcap
+    config-opts:
+      - --enable-ipv6
+    sources:
+      - type: archive
+        url: "https://www.tcpdump.org/release/libpcap-1.9.0.tar.gz"
+        sha256: 2edb88808e5913fdaa8e9c1fcaf272e19b2485338742b5074b9fe44d68f37019
+    cleanup:
+      - /bin
+
+  - name: v4l2-utils
```

Comment:
libv4l already exists in the freedesktop runtime. Is there a reason why you'd build them here?